### DISCUSSION TITLE
How check Coral AI working

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#1909] [Ask A Question]
Hi!

I have USB Coral Ai but frigate.detector.cpu is eating cpu resources. How check that Coral actually working?

And using only one usb coral with RC3

At config.yml beginning:

detectors:
  coral:
    type: edgetpu
    device: usb

edit: indentions are ok but cannot get those right in here

**Comment by McPaHa** [comment]
Damn... noe edgetpu detected...

Edit: problem solved when moving usb-detector from USB-hub and attached that directly to Intel Nuc usb port.

**Comment by McPaHa** [comment]
LOL At least it somehow working with old Intel Nuc and cpu based detector and 4 Full HD cameras... ;)

**Comment by svo08** [comment]
Hi,
which Generation of NUC/CPU you are exactly using with 4 Full HD Cameras ?
Currently im using 4th Gen NUC i5 4250U and Google Coral USB.
But NUC was nearly at 100% with 3 CAMs (High Resulation Cams).
Maybe it will help a little bit when switching to Second Stream with LOW RES.
Not sure if can manage to add all 6 CAMs.

**Reply by blakeblackshear** [reply]
Frame rate matters a lot too.

**Reply by McPaHa** [reply]
I have Nuc D54250WYK. Four FullHD cameras with 5fps.

I am not saying that this is ok. Cpu is almost every time ~ 95% and load average 5.xx but I didn't notice my missing Coral adapter anywhere but checking htop and portainer log. With Coral.ai cpu load is much lower.

Anyway I am planning to buy newer nuc or similar because there are other purposes also what I am doing with NUC.
